magenta/models/rl_tuner/rl_tuner_ops.py

```diff
@@ -21,6 +21,7 @@
 
 import numpy as np
 import tensorflow as tf
+from tensorflow.contrib.rnn import rnn_cell
```

Comment:
I should have noticed that was your preferred style. I've made the change: now, do you prefer force-pushing the fixup to keep the PR clean or add an additional commit on top?